### PR TITLE
py-pyseer and deps: new package

### PR DESCRIPTION
**Pull request by alex391**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-glmnet-python/package.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-glmnet-python/package.py]:
```suggestion
# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

File: var/spack/repos/builtin/packages/py-pyseer/package.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-pyseer/package.py]:
```suggestion
# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
```

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed homepages and sha256s as well as the dependencies.  The copyright year does not technically have to be up-to-date *this* year but recommend doing so anyway.

@adamjstewart What are your thoughts on this PR (esp., `py-glmnet-python` when have different `py-glmnet`)?

**Review by adamjstewart — DISMISSED** [review]
I agree with all name choices, I would also pick these names.

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**